components/SpotlightMenu/SpotlightMenuListItem.tsx

```diff
@@ -40,7 +40,7 @@ const SpotlightMenuListItem: FunctionComponent<ComponentProps> = ({ title, child
         })}
         onClick={toggleActive}
       >
-        {title}
+        {title?.toLowerCase()}
```

Comment:
I'd rather you don't be so sure that there's a title ... there's a reason why it was made optional.

Reply:
Made the changes for the title definition above